src/ngraph_builder.cc

```diff
@@ -1724,12 +1724,29 @@ static Status TranslateFusedBatchNormOp(
     ng_y = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 0);
     ng_mean = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 1);
     ng_variance = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 2);
+    // This is for Bessel's correction in ng_variance:
```

Comment:
Bessel correction is just `n/(n-1)` multiplied by variance
In batchnorm we compute variance over teh batchsize. So `n` is most probably batchsize.

Could you please provide the link to the code that you are refering to when you make this correction, so that we have a better understanding.

Reply:
The `n` is input data size divided by the channel size and not equal to n. For large input data size, this correction is negligible. 
Some of example implementation in TensorFlow can be found here: https://github.com/tensorflow/tensorflow/blob/a8e78e2e617b6ca10f4878fe99fdf43ddedfa7c6/tensorflow/core/kernels/mkl_fused_batch_norm_op.cc#L193